test/integration/associations/belongs-to-many.test.js

```diff
@@ -1277,7 +1483,7 @@ describe(Support.getTestDialectTeaser('BelongsToMany'), () => {
       ]);
 
       const result = await article.hasLabels([label]);
-      await expect(result).to.be.false;
```

Comment:
Why remove await here?

Reply:
My answer is a question here :) why do we need to have await here as this is just a test case last validation step and doesn't need to be asynchronous? Am I right?